### ISSUE TITLE
Neuropathic pain

### ISSUE THREAD

**Issue by andrewdevereau** [obophenotype/human-phenotype-ontology#3986]
submitted on behalf of David Bennett, Professor of neurology and neurobiology, Nuffield department of clinical neurosciences, Uni. of Oxford

_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Neuropathic pain

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
Neuropathic pain is pain arising as a direct consequence of a lesion or disease affecting the somatosensory system. 
PMID: 18003941
PMID: 27115670

### 4. Parent term (use [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
Chronic pain HP:0012532

### 5. Which diseases are characterized by this term ? (e.g. Orphanet- or OMIM-id)
ORPHA:324585  Autosomal dominant intermediate Charcot-Marie-Tooth disease with neuropathic pain

### 6. Your nano-attribution (ORCID-id or label, e.g. _HPO:probinson_ (organization:name))
https://orcid.org/0000-0002-1089-1543

**Comment by pnrobinson** [comment]
Sorry for the late response.
The HPO includes terms for phenotypes but avoids encoding inferences about the etiology.
Neuropathic pain includes a number of components (The wikipedia description is good: https://en.wikipedia.org/wiki/Neuropathic_pain), and the HPO has terms to cover these, e.g., Allodynia (https://hpo.jax.org/app/browse/term/HP:0012533) and Dysesthesia (https://hpo.jax.org/app/browse/term/HP:0012534). 
The reason for not including etiology in the terms is that if we started down this road we would have to include multiple versions of terms to encode the (assumed) aetiologies; for instance, type I diabetec hyperglycemia, type II diabetic hyperglycemia, steroid-induced hyperglycemia,...
Also there is a risk that either the ontology would be wrong or the assumptions of the people coding patients with the ontology would be wrong leading to false inferences.
So I would advice coding patients or diseases with the features (such as Allodynia) that led to the inference of Neuropathic pain.

**Comment by andrewdevereau** [comment]
Peter – I’m not sure if this is the best way to do this but I would like to bring David Bennett into the discussion of this proposed HPO change. David raised the original issue and would like to follow it up with you.

Best wishes

Andrew

**Comment by pnrobinson** [comment]
Hi Andrew, please forward my email to David or send me an email to settle on a mode of communication!
